.github/workflows/deploy_prd.yml

```diff
@@ -0,0 +1,85 @@
+name: Manual Production Deploy
+
+concurrency: Manual Production Deploy
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version'
+        required: true
+
+jobs:
+  deploy_to_production_start_slack:
+    if: github.repository_owner == 'Informatievlaanderen'
+    name: Deploy to production started
+    environment: prd
```

Comment:
In municipality registry we use newprd. Does it matter? https://github.com/Informatievlaanderen/municipality-registry/blob/main/.github/workflows/CD%20New%20Production.yml#L14C18-L14C24